nautobot/core/templates/template.css

```diff
@@ -0,0 +1,87 @@
+{% load static %}
```

Comment:
I have to put this in the core/template subfolder so that the django template engine can find it.
[Here, I prepended a 'css/' prefix to the url, but this is not necessary, and you don't need a "css" folder in your project; just make sure that the template engine can find "home_global.css"; that is, put it in the /template/ subfolder of any installed app, or even in the project if it is listed among the installed apps:](https://stackoverflow.com/questions/55144698/use-django-template-syntax-in-a-css-file/55225412#55225412)